src/main/kotlin/com/openlattice/launchpad/configuration/IntegrationTables.kt

```diff
@@ -20,7 +21,9 @@ class IntegrationTables {
          * 3. table_name
          * 4. start
          */
-        const val LOG_INTEGRATION_STARTED  = "INSERT INTO integration_activity (integration_name, host_name, table_name, start) VALUES (?,?,?,?)"
+        const val LOG_INTEGRATION_STARTED  = "INSERT INTO integration_activity (integration_name, host_name, table_name, start) " +
```

Comment:
would kotlin multiline strings be better here?
```
val ... = """
    INSERT INTO ...
    VALUES (?, ?, ?, ?)
"""
```

Reply:
100% did not know this was a thing but will definitely investigate for the future of queries

Reply:
Yep, gotta say I am a fan of multi-line strings now. Any changes will be in a separate PR b/c this is strictly a `merge develop to master` for release PR